cmd/bee/cmd/hasher.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2021 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/bcrypt"
+)
+
+func (c *command) initHasherCmd() (err error) {
+	cmd := &cobra.Command{
+		Use:   "hasher",
```

Comment:
maybe just `bcrypt`? hasher has some historical connotation related to file-hashing

Reply:
fixed